packages/context-entries/lib/Keys.ts

```diff
@@ -281,6 +281,13 @@ export const KeysRdfUpdateQuads = {
   destination: new ActionContextKey<IDataDestination>('@comunica/bus-rdf-update-quads:destination'),
 };
 
+export const KeysMergeBindingsContext = {
+  /**
+   * The data source of the binding
```

Comment:
Name and comment is a bit inaccurate, since we have a `string[]`, so multiple sources.